File: HB/pysrc/프로그래머스/레벨0/Day11/합성수_찾기.py
def solution(n):
    answer = 0
    cnt = 0
    if n == 1 or n == 2 or n == 3:
        answer = 0
        
    else:
        for i in range(2, n+1):
            cnt = 0
            for j in range(2,i):
                if i%j == 0:
                    answer += 1
                    break
    return answer

File: SB/src/프로그래머스/레벨0/Day13/합성수_찾기.java
package 프로그래머스.레벨0.Day13;

public class 합성수_찾기 {
    class Solution {
        public int solution(int n) {
            int answer = 0;
            int cnt = 0;
            for (int i = 1; i <= n; i++) {
                cnt = 0;
                for (int j = 1; j <= n; j++) {
                    if (i % j == 0)
                        cnt ++;
                }
                if (cnt >= 3)
                    answer++;
            }
            return answer;
        }
    }
}
